sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 2.0.0-beta.3 (Unreleased)
 
+- Removed `authenticationRecord`, `disableAutomaticAuthentication` and `authenticate()` from the credential `UsernamePasswordCredential`. While MSAL does support this, allowing `authenticationRecord` arguably could result in users authenticating through an account other than the one they're specifying with the username and the password.
```

Comment:
Lets have this in a separate section `Breaking changes from 2.0.0-beta.1`

Reply:
Done!